codersdk/healthsdk/healthsdk.go

```diff
@@ -212,24 +252,19 @@ type ProvisionerDaemonsReportItem struct {
 	Warnings                   []health.Message `json:"warnings"`
 }
 
+// WebsocketReport shows if the configured access URL allows establishing WebSocket connections.
 type WebsocketReport struct {
 	// Healthy is deprecated and left for backward compatibility purposes, use `Severity` instead.
-	Healthy   bool            `json:"healthy"`
-	Severity  health.Severity `json:"severity" enums:"ok,warning,error"`
-	Warnings  []string        `json:"warnings"`
```

Comment:
review: this is technically a breaking change... but it should have been this way from the beginning